db/migrate/20160502231403_create_orders.rb

```diff
@@ -0,0 +1,16 @@
+class CreateOrders < ActiveRecord::Migration
+  def change
+    create_table :orders do |t|
+      t.string :card_name
+      t.string :email
+      t.string :address
+      t.integer :credit_card
+      t.string :exp_date
+      t.integer :cvv
+      t.string :zip
```

Comment:
nice job making the zip code a string! Weird zip codes that start with zero... 😾  0️⃣